frontend/src/pages/SubmitProperty.tsx

```diff
@@ -32,16 +32,12 @@ const SubmitProperty = (props: any) => {
     state => state.parcel.parcelDetail as IParcelDetail,
   );
   const properties = useSelector<RootState, IProperty[]>(state => state.parcel.parcels);
-  const addParcelRequest = useSelector<RootState, IGenericNetworkAction>(
```

Comment:
this logic and the removed logic below were moved to the form where formik values are still in context.